### PR TITLE
Remove test break due to randomly generating a file with zero results.

### PR DESCRIPTION
**Pull request by michaelcfanning**
@shaopeng-gh @yongyan-gh @EasyRhinoMSFT

### CODE REVIEW

**Comment by shaopeng-gh** [comment]
The `dotnet format` error is spaces:
D:\a\sarif-sdk\sarif-sdk\src\Test.UnitTests.Sarif\Core\SarifLogTests.cs(153,1): error WHITESPACE: Fix whitespace formatting. Replace 26 characters with '\r\n\s\s\s\s\s\s\s\s\s\s\s\s'. [D:\a\sarif-sdk\sarif-sdk]

**Review by shaopeng-gh — APPROVED** [review]
:shipit:

**Review by yongyan-gh — APPROVED** [review]
:shipit:

File: src/Test.UnitTests.Sarif/Core/SarifLogTests.cs [hunk 1]
```diff
@@ -149,7 +149,13 @@ public void SarifLog_SplitPerTarget()
         public void SarifLog_SplitPerTarget_WithEmptyLocations()
         {
             Random random = RandomSarifLogGenerator.GenerateRandomAndLog(this.output);
-            SarifLog sarifLog = RandomSarifLogGenerator.GenerateSarifLogWithRuns(random, 1);
+            SarifLog sarifLog = null;
+
```
Comment (yongyan-gh) [inline src/Test.UnitTests.Sarif/Core/SarifLogTests.cs:153]: extra whitespaces

**Pull request merged**